src/ace/managers/copper.c

```diff
@@ -57,19 +57,285 @@ void copSwapBuffers(void) {
 	pCopList->pBackBfr = pTmp;
 }
 
+#ifdef ACE_DEBUG
+static const char* _custom_register_names[256] = {
```

Comment:
we use CamelCaps for names, preceded with prefix.

- Since it's an array, it is a pointer, hence the `p` prefix.
- Since it's the `static` global var, additional `s_` prefix is added.

So, the final name should be `s_pCustomRegNames` or something.

Also, since this array is used only in `copDumpCmd()`, perhaps add it as `static const` var inside fn? If so, skip the `s_` prefix.